lang/octave/octave.def

```diff
@@ -0,0 +1,11 @@
+Bootstrap: library
+From: ubuntu:18.04
```

Comment:
Add a help section to describe what does this container and which version of Octave it's supported